### DISCUSSION TITLE
Vercel部署个人博客 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#165] [General]
# Vercel部署个人博客 - 愧怍的小站

image-20220511170700075

https://kuizuo.cn/vercel-deploy-blog/#comments

**Comment by xyhcode** [comment]


**Reply by kuizuo** [reply]
看你的域名是哪里买的了，如果是阿里云或腾讯云等国内的服务商，还需要在对应的服务商购买一个云服务器或轻量应用服务器才能提交相关备案信息。

还有，只需要ICP备案就能让别人通过域名来访问站点了，至于公安备案不是必须，这需要根据你站点所属性质来决定，比如博客站点其实就没必要到当地的公安去备案了。

vercel服务器当然是国外的，但是服务器只是给你使用，不代表是你的，所以备案条件下还是需要有一台服务器的（国内是这样的），至于国外服务商我是没购买过，也没具体接触过。所以不大好做回复。

总之国内ICP备案还是比较繁琐的

**Comment by xyhcode** [comment]
国内服务器我要有 然后我的域名备案了ICP的 域名绑在了vercel上 所以我想问问要不要公安备案

2022年8月29日 12:41，愧怍 ***@***.******@***.***>> 写道：



看你的域名是哪里买的了，如果是阿里云或腾讯云等国内的服务商，还需要在对应的服务商购买一个云服务器或轻量应用服务器才能提交相关备案信息。

还有，只需要ICP备案就能让别人通过域名来访问站点了，至于公安备案不是必须，这需要根据你站点所属性质来决定，比如博客站点其实就没必要到当地的公安去备案了。

vercel服务器当然是国外的，但是服务器只是给你使用，不代表是你的，所以备案条件下还是需要有一台服务器的（国内是这样的），至于国外服务商我是没购买过，也没具体接触过。所以不大好做回复。

总之国内ICP备案还是比较繁琐的

**Reply by kuizuo** [reply]
公安备案不是必须的，不备案网站也是能正常上线的，我其实有写过一篇关于 [服务器与域名备案](https://kuizuo.cn/server-and-domain-beian#%E5%85%AC%E5%AE%89%E5%A4%87%E6%A1%88) ，你可以了解下我当时备案的过程

**Reply by kuizuo** [reply]
而且公安备案很麻烦，可能还需要你去联系当地的公安网安服务人员，最主要还是看你网站是什么性质。像博客其实就完全没必要了，像很多网站都没有悬挂公安备案号，但ICP备案一般是要悬挂的，不然有可能定期会发短信提醒您，吧唧吧唧的

**Comment by xyhcode** [comment]
OK 懂了 我的是个人博客 谢谢了

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: 愧怍 ***@***.***>
发送时间: Monday, August 29, 2022 12:51:16 PM
收件人: kuizuo/blog ***@***.***>
抄送: 羡羡 ***@***.***>; Comment ***@***.***>
主题: Re: [kuizuo/blog] Vercel部署个人博客 - 愧怍的小站 (Discussion #165)


而且公安备案很麻烦，可能还需要你去联系当地的公安网安服务人员，最主要还是看你网站是什么性质。像博客其实就完全没必要了，像很多网站都没有悬挂公安备案号，但ICP备案一般是要悬挂的，不然有可能定期会发短信提醒您，吧唧吧唧的

―

**Comment by orangelegy** [comment]
666

**Comment by iampapertiger** [comment]
如何进入blog的管理后台呢？不会每次编辑，重新部署吧。。。

**Reply by kuizuo** [reply]
没错，你还真回答对了。

我的编写文章习惯是在 notion 上，然后导出成 md 文件到 blog 目录，然后 git commit 提交记录，借助 github actions自动构建与部署，便可完成网站的更新，即文章的上传。因此实际麻烦的操作是在 **notion 导出和 git 提交上**，但这点麻烦是在我可接受的范围内。

> 至于说为什么选 notion，是因为我有把文章同步到其他平台的需求。

我一开始的打算就没想写一个 blog 的后台管理，因为文章数据动态存储的话，就需要通过二次请求来获取文章数据，这远不如直接直接打包成纯静态页面来的高效。并且你打算这么做的话，你还需要考虑 ssr 渲染与 seo，不然你的文章就别指望被搜索引擎收录。

当然每个人的需求都不同，只是我认为这种编写 blog 的方式是比较舒服的。

**Reply by iampapertiger** [reply]
谢谢老哥，你的审美真的很在线，最喜欢的个blog风格了~~

**Comment by 1226085293** [comment]
博主你好，我使用Netfily部署了我的github博客，为什么国内无法访问呢？

**Reply by kuizuo** [reply]
难不成，netlify.app 也被墙了？能否提供相关截图？

**Reply by 1226085293** [reply]
我用 netlify 部署的我的 github page 的网站，部署成功了，但是只有开 vpn 才能进，试过电脑手机不开 vpn 都进不了网站，netlify 站点我已经删了

**Reply by 1226085293** [reply]
难道是必须用 gitee 才行？但是 gitee 也有 gitee pages，netlify 就没什么意义了

**Reply by kuizuo** [reply]
我这边测试国内网络无法访问，需要梯子。也就是说 netlify.app 不出意外和 vercel.app 一样被墙了。

要把 web 应用部署到这两个平台上并供国内用户访问的话，目前应该就需要搞一个域名，然后将域名 cname 到这些平台上。
